server/startup/migrations/v238.ts

```diff
@@ -0,0 +1,34 @@
+import { Mongo } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations/server';
+import { isEnterprise } from '../../../ee/app/license/server';
```

Comment:
We should never import the EE folder directly.

Reply:
done